src/prometheus_fastapi_instrumentator/middleware.py

```diff
@@ -102,7 +103,10 @@ async def send_wrapper(event: ASGISendEvent) -> None:
         except Exception as exc:
             raise exc
         finally:
-            status = str(status_code)
+            if type(status_code) == HTTPStatus:
+                status = str(status_code.value)
+            else:
+                status = str(status_code)
```

Comment:
You should probably use `isinstance` here instead of checking the type specifically:

```suggestion
            status = str(status_code.value) if isinstance(status_code, HTTPStatus) else str(status_code)
```

https://switowski.com/blog/type-vs-isinstance/
https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance

Reply:
Thanks for the suggestion!